portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -64,7 +66,7 @@ public String getName() {
 
   // all options: "newest (descending), oldest (ascending), alphabetical, reverse-alphabetical"
   public String sort = "newest";
-  public ArrayList<String> messages = new ArrayList<String>(List.of("Hello", "Goodbye", "Thanks"));
+  // public ArrayList<String> messages = new ArrayList<String>(List.of("Hello", "Goodbye", "Thanks"));
```

Comment:
Please keep the code clean.  This code is already in revision history is you want it back later.  Or you can copy code blocks elsewhere as backup.

Reply:
Github is dumb.  I can't leave a comment for the if,else,else block down there.  I think Java has switch-case which can probably make it more readable.  No big deal though.